tests/swo/mpt/sync/test_syncer.py

```
@@ -494,3 +495,96 @@ def test_get_pma(agreement, syncer):
     result = syncer.get_pma(agreement)
 
     assert result == "651706759263"
+
+
+def test_sync_subscriptions_creates_new(  # noqa: WPS218
```

Ruff (0.15.17)

[warning] Invalid rule code in `# noqa`: WPS218

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)